src/test/resources/features/attachments.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Adding attachments
+
```

Comment:
check my reviews at https://github.com/openmrs/openmrs-contrib-qaframework/pull/3/files and work on this similarly